src/platform_impl/windows/dark_mode.rs

```diff
@@ -181,6 +191,71 @@ fn should_apps_use_dark_mode() -> bool {
     .unwrap_or(false)
 }
 
+// This enables dark mode menus
+fn set_preferred_app_mode(is_dark_mode: bool) -> bool {
+  let build_version = (*WIN10_BUILD_VERSION).unwrap();
+
+  // SetPreferredAppMode is for Windows 1903+ (18362)
+  if build_version >= 18362 {
+    type SetPreferredAppMode = unsafe extern "system" fn(PreferredAppMode) -> bool;
+    lazy_static! {
```

Comment:
This function doesn't feel like really need to be a static one. Could it just be code block?

Reply:
I just copied the pattern used in `should_apps_use_dark_mode`, making these static makes more sense since the value returned by `GetProcAddress` won't change.